docs/docs/docs_es/introduction/getting-started.mdx

```diff
@@ -0,0 +1,128 @@
+---
+title: Comencemos
+---
+
+
+## Create React App
+Recoil es una librería para el manejo del estado de React, y por lo tanto necesitas tener React instalado y corriendo para poder usarlo. La forma más fácil y recomendada para usar una aplicación con React, es instalando el proyecto [Create React App](https://github.com/facebook/create-react-app#creating-an-app):
+
+```shell
+npx create-react-app my-app
+```
+
+> [`npx`](https://medium.com/@maybekatz/introducing-npx-an-npm-package-runner-55f7d4bd282b) es una herramienta para correr paquetes que viene con npm 5.2+ o en cualquier versión más reciente, leer las [instrucciones para versiones viejas de npm](https://gist.github.com/gaearon/4064d3c23a77c74a3614c498a8bb1c5f).
+
+Para ver más formas de como instalar el proyecto de Create React App, mira la [documentación oficial](https://github.com/facebook/create-react-app#creating-an-app).
+
+## Instalación
+
+Recoil está disponible en <a href="https://www.npmjs.com/get-npm" target="_blank">npm</a>. Para instalar la última versión estable ejecuta el siguiente comando:
+
+```shell
+npm install recoil
+```
+
+O si usas <a href="https://classic.yarnpkg.com/en/docs/install/" target="_blank">yarn</a>:
+
+```shell
+yarn add recoil
+```
+
+## RecoilRoot
+
+Los componentes que usan el estado de Recoil, necesitan tener el `RecoilRoot` en algún lugar del árbol padre. Un buen lugar para poner esto sería en tu componente principal:
+
+
+```jsx
+import React from 'react';
+import {
+  RecoilRoot,
+  atom,
+  selector,
+  useRecoilState,
+  useRecoilValue,
+} from 'recoil';
+
+function App() {
+  return (
+    <RecoilRoot>
+      <CharacterCounter />
+    </RecoilRoot>
+  );
+}
+```
+
+Vamos a implementar el componente `CharacterCounter` en la siguiente sección.
+
+## Átomo
+
+Un átomo representa una pieza del estado. Los átomos pueden ser leídos y actualizados desde cualquier componente. Los componentes que leen valores de un átomo son implícitamente suscritos a ese átomo, por lo tanto, cada cambio en el átomo renderizará todos esos componentes suscritos.
```

Comment:
pieza->parte, replace comma by dot before por lo tanto, re-renderizarán

Reply:
I used re-renderizará in place of re-renderizarán, please let me know if this is correct.

Reply:
You’re right, oops